gui/windows/main_window.py

```diff
@@ -416,10 +416,7 @@ def load_npk(self, path: str):
 
         def _load_entries():
             for i in range(npk_file.file_count):
-                x = npk_file.read_entry(i)
-                if x.data_flags & NPKEntryDataFlags.ERROR:
```

Comment:
This is removed because the errors are already logged when the flag is added. And if the loop terminated without cancel flag set, it'll hang the program.